template.json

```diff
@@ -111,43 +127,52 @@
                     },
                     "stun": {
                         "value": 20,
-                        "max": 20
+                        "max": 20,
+                        "core": 20
                     },
                     "end": {
                         "value": 20,
-                        "max": 20
+                        "max": 20,
+                        "core": 20
                     },
                     "size": {
                         "value": 0,
                         "max": 0
                     },
                     "running": {
                         "value": 12,
-                        "max": 12
+                        "max": 12,
+                        "core": 12
                     },
                     "swimming": {
                         "value": 4,
-                        "max": 4
+                        "max": 4,
+                        "core": 4
                     },
                     "leaping": {
                         "value": 4,
-                        "max": 4
+                        "max": 4,
+                        "core": 4
                     },
                     "flight": {
                         "value": 0,
-                        "max": 0
+                        "max": 0,
+                        "core": 0
                     },
                     "swinging": {
                         "value": 0,
-                        "max": 0
+                        "max": 0,
+                        "core": 0
                     },
                     "teleportation": {
                         "value": 0,
-                        "max": 0
+                        "max": 0,
+                        "core": 0
                     },
                     "tunneling": {
                         "value": 0,
-                        "max": 0
+                        "max": 0,
+                        "core": 0
```

Comment:
Looks like we also need the other movement powers. I appear to have forgotten to add gliding but will also need a way of taking it out if it's 6e since it's a 5e only movement type. See `HERO.movementPowers` and `HERO.movementPowers5e` if all movement types are required.

Reply:
Need this for gliding fix so going to merge and do gliding as separate work.